utils/regr/moda.cpp

```diff
@@ -186,7 +187,7 @@ mcsv1_UDAF::ReturnCode moda::init(mcsv1Context* context,
         {
             context->setColWidth(8);
         }
-        else
+        else if (utils::widthByPrecision(colTypes[0].precision))
```

Comment:
Shouldn't the returned value here be compared against datatypes::MAXDECIMALWIDTH?

Reply:
It should be a different method.